scripts/deploy_providers_and_joint.py

```diff
@@ -0,0 +1,74 @@
+from brownie import Contract, SpookyJoint, accounts, config, ProviderStrategy, network
+
+def main():
+
+    acct = accounts.add(config["accounts"][0])
+    
+    # USDC
+    tokenA = Contract("0x04068DA6C83AFCFA0e13ba15A6696662335D5B75")
+    vaultA = Contract("0xEF0210eB96c7EB36AF8ed1c20306462764935607")
+    # WFTM
+    tokenB = Contract("0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83")
+    vaultB = Contract("0x0DEC85e74A92c52b7F708c4B10207D9560CEFaf0")
+
+    # Provider strats
+    if network.show_active() == "ftm-main":
+        provider_strat = acct.deploy(ProviderStrategy, vaultA, publish_source=True)
+    else:
+        provider_strat = acct.deploy(ProviderStrategy, vaultA)
+
+    print(f"Original provider strat deployed to {provider_strat.address}")
+    providerA = ProviderStrategy.at(provider_strat.clone(vaultA).return_value)
```

Comment:
are you cloning the right strategy?

Reply:
or you would need to deploy the first one

Reply:
I am deploying an original Provider -> provider_strat and then clone that one twice. I could use the original as the providerA or providerB but I like the original to remain unused

Reply:
coolio